tests/test_e2e.py

```diff
@@ -146,24 +142,7 @@ def test_vllm_e2e(self, protocol, setup_and_teardown):
         utils.wait_for_server_ready(timeout=300)
 
         self._infer(model, prompt=PROMPT, protocol=protocol)
-        self._profile(model, protocol=protocol)
-
-    @pytest.mark.parametrize("protocol", ["grpc", "http"])
-    def test_non_llm(self, protocol, setup_and_teardown):
```

Comment:
Why is this test removed? Can we keep it in for the `infer()` part even if we don't do `profile()`? Want to keep some sanity test that `triton infer` is working on arbitrary models.

Reply:
Sure, let me return this. This was based on our conversations about Triton CLI not officially supporting non-LLM inference at the moment. I thought I added this test and missed that it's being removed in my review of the PR.